kafka/adminapi.go

```diff
@@ -331,8 +331,10 @@ func (a *AdminClient) waitResult(ctx context.Context, cQueue *C.rd_kafka_queue_t
 				}
 
 			default:
-				// Wait for result event for at most 100ms
-				rkev := C.rd_kafka_queue_poll(cQueue, 100)
+				// Wait for result event for at most 50s
```

Comment:
I believe this should be in milliseconds not seconds. A 50 second delay in context cancellation would likely induce saddness.